kuma/javascript/src/icons/close.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="28" viewBox="0 0 24 28"><path d="M20.281 20.656c0 0.391-0.156 0.781-0.438 1.062l-2.125 2.125c-0.281 0.281-0.672 0.438-1.062 0.438s-0.781-0.156-1.062-0.438l-4.594-4.594-4.594 4.594c-0.281 0.281-0.672 0.438-1.062 0.438s-0.781-0.156-1.062-0.438l-2.125-2.125c-0.281-0.281-0.438-0.672-0.438-1.062s0.156-0.781 0.438-1.062l4.594-4.594-4.594-4.594c-0.281-0.281-0.438-0.672-0.438-1.062s0.156-0.781 0.438-1.062l2.125-2.125c0.281-0.281 0.672-0.438 1.062-0.438s0.781 0.156 1.062 0.438l4.594 4.594 4.594-4.594c0.281-0.281 0.672-0.438 1.062-0.438s0.781 0.156 1.062 0.438l2.125 2.125c0.281 0.281 0.438 0.672 0.438 1.062s-0.156 0.781-0.438 1.062l-4.594 4.594 4.594 4.594c0.281 0.281 0.438 0.672 0.438 1.062z"/></svg>
```

Comment:
You can remove the `width` and `height` attributes here as the `viewBox` is perfect on its own. I would also suggest that you add `role="presentation"` for accessibility. You then have two options with regards to giving this a title.

1. Add an `aria-label` attribute, with a text string such as "Close dialog", to the anchor element or button that wraps this. Then also add an `id` to the same element. Then, add an `aria-labelledby` to the SVG and set its value to that of the `id`.
2. Just before the first `path` element add `<title id="close-icon-title">Close dialog</title>`. On the SVG tag add `aria-labelledby="close-icon-title"`.

Reply:
This is perfect feedback. I knew I was leaving this out, but didn't know the right way to handle it. Thanks.

Reply:
Since I'm using role="presentation", I'd expect that to mean that the svg will just be ignored and not need to be labelled. So can't I just put the aria-label on the button and leave it at that?  (I'm updating the PR to do that, but you can tell me if I really need the id and aria-labelledby

Reply:
Sure, if the button has `aria-label` that will do. Thanks @davidflanagan